modules/standard/IO.chpl

```diff
@@ -5177,6 +5177,62 @@ proc _channel.writebits(v:integral, nbits:integral) throws {
   try this.write(new ioBits(v:uint(64), nbits:int(8)));
 }
 
+/*
+   Write ``numBytes`` of data from a :class:`~CTypes.c_ptr` to a fileWriter
+
+   Note that native endianess is always used.
+
+   If ``numBytes`` is not evenly divisible by the size of ``t``, then the
```

Comment:
Is "evenly divisible" what we actually mean here? From the argument descriptions I assumed it should be "if `numBytes` is less than the size of `t`"

Reply:
"evenly divisible" is what I meant, but you're right that it isn't very clear. I added an example to describe why this is the case. Lmk if it makes more sense.